src/index.js

```diff
@@ -36,7 +37,32 @@ module.exports = {
     }
   },
   create: {
-    handlers: () => ({ filename: 'index.ts', body })
+    handlers: ({ lambda: { pragma } }) => {
+      let body
+      switch (pragma) {
+      case 'http':
+        body = httpHandlerBody
+        break
+      case 'queues':
+        body = sqsHandlerBody
+        break
+      case 'events':
+        body = snsHandlerBody
+        break
+      case 'scheduled':
+        body = scheduledHandlerBody
+        break
+      case 'tables-streams':
+        body = dynamoDbStreamHandlerBody
+        break
+      case 'ws':
+        body = websocketHandlerBody
+        break
+      default:
+        throw Error(`Unknown lambda type: ${pragma}`)
```

Comment:
Hmm, thinking we should support `customLambdas` via the `set` API, I think right now this would be a regression? See: https://arc.codes/docs/en/guides/plugins/set#set.customlambdas

Entirely subjective but I personally find switches to be kind of ugly and overly verbose; this could just as well be matching destructured property names, as in:
```js
let { dynamoDbStreamHandlerBody: tablesstreams... } = require('./handlers')
// ...
// create.handlers:
let normalizedPragma = pragma.replace(/-/, '').toLowerCase()
if (handlers[normalizedPragma]) body = handlers[normalizedPragma]
```

Reply:
Totally agree on the switch statements. Ive traded it out for a hash of pragma: handler.
Ive also added a default "custom" lambda.